build.gradle

```diff
@@ -67,9 +71,9 @@ jar {
                    "Implementation-Version": VERSION_NAME,
                    "Implementation-Vendor": VENDOR_NAME,
                    "Bundle-SymbolicName": POM_ARTIFACT_ID,
-                   "Export-Package": 'com.stripe.*')
+                   "Export-Package": "com.stripe.*")
 
-        version = VERSION_NAME
+        archiveVersion = VERSION_NAME
```

Comment:
ooc what is this change for?

Reply:
`version` is deprecated and replaced by `archiveVersion`.